Hmwk#4_CSV/readcsv.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import csv
+import os
+import sys
+import re
+from typing import List, Union
+
+#--filename    -f    Path to a file containing a list of users to scrape.
+#--columnname   -f    name of column to read from csv file.
+
+def getArgs(argv = None): 
+    parser = argparse.ArgumentParser(description='Add filepath to file to read from.') #add arguments
+    parser.add_argument('-f','--filepath', type=str,help='path to file needed to read')
+    parser.add_argument('-c','--columnname', type=str,help='column from csv file needed to read')
```

Comment:
If you write "-filepath" or "--filepath", it means that **filepath** is optional, and your program can work without it.

Use positional argument`parser.add_argument('filepath', type=str, help='path to file needed to read')` if an argument is required.